.github/workflows/macos.yml

```diff
@@ -71,25 +71,31 @@ jobs:
 
       - run: make
 
+      - name: Set test options for skipped tests
+        run: |
+          set -ex
```

Comment:
I saw that you mentioned that the `set -x` is good enough for `set -ex` in another PR. So, let me check on my local and fix it with the `set -x`.

Reply:
OK. I just confirmed that the `set -x` checked the exit status non-zero by the `false` command on my forked repository.

```
- name: something
  run: |
    set -x
    false
    a_command
```

Reply:
I fixed it.